bin/propolis-standalone/src/main.rs

```diff
@@ -959,8 +959,13 @@ fn generate_smbios(params: SmbiosParams) -> anyhow::Result<smbios::TableBytes> {
     Ok(smb_tables.commit())
 }
 
-fn generate_bootorder(config: &config::Config) -> anyhow::Result<fwcfg::Entry> {
-    let names = config.main.boot_order.as_ref().unwrap();
+fn generate_bootorder(
+    config: &config::Config,
+) -> anyhow::Result<Option<fwcfg::Entry>> {
+    eprintln!("bootorder declared as {:?}", config.main.boot_order.as_ref());
```

Comment:
This should probably pass in the `slog::Logger`  and emit it as something like debug level?

Reply:
(i tend to lean on eprintln/panic in Rust as easily-findable "reconsider this and make it worth actually landing" sigils. here and in other places it'll probably get sent through to `slog` if relevant bits aren't otherwise reworked)

Reply:
Sounds good